torch/_inductor/codegen/common.py

```diff
@@ -412,6 +412,19 @@ def make_inplace(self, input_name, output_name):
             self.inplace_buffers[input_name] = buf
             self.inplace_buffers[output_name] = buf
 
+    def seed_offset(self, name, value):
+        if "load_seed_offset" in self.sizevars.values():
```

Comment:
do we have any testcases for multiple load_seed_offset args?

Reply:
Also, this wouldn't work if you change `name` at the callsite

Reply:
I changed the hardcode string to the `name` in https://github.com/pytorch/pytorch/pull/102104/commits/e6268cf9d7b85fc615503dfe642582b653dac1a1 . 

The simple example for multiple load_seed_offset could be the following.
```
def fn():
    random_tensor1 = torch.randint(10, [1024], device="cuda")
    random_tensor2 = torch.randint(11, [1024], device="cuda")
    random_tensor3 = torch.randint(10, [1024], device="cuda")
    tensor4 = random_tensor1 + random_tensor2 +random_tensor3
    return tensor4
```

Reply:
I'm thinking that there is another hardcode string in [divisible_by_16](https://github.com/pytorch/pytorch/pull/102104/files#diff-ea3834aeda2e22f49dd5ba0becd456b449b10d17dfe64c34e89072701c82873dR82) check. It's better to replace it too. Do you know where I should put this hardcode string as a global string?

Reply:
Yeah I understand examples for multiple load_seed_offset exist, I'm asking if we have a test for those

Reply:
I only tested several models. not found multiple load_seed_offset for now.

Reply:
Do you mean adding a unit test?

Reply:
Yes, please add a unit test for this if it doesn't exist.

Reply:
add a unit test by https://github.com/pytorch/pytorch/pull/102104/commits/dc1e18705adef2ee2ce8effd67aed20e1e7aeb4d